packages/suite/src/utils/suite/env.ts

```diff
@@ -145,13 +145,21 @@ export const setOnBeforeUnloadListener = (callback: () => void) => {
     window.addEventListener('beforeunload', callback);
 };
 
-export const getOsTheme = (): SuiteThemeVariant => {
-    if (typeof window === 'undefined') return 'light'; // in SSR, where window object is not defined, just return light theme
-    // retrieving os color scheme is supported in Chrome 76+, Firefox 67+
-    if (window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches) {
-        return 'dark';
-    }
-    return 'light';
+const getDarkThemeQuery = (): MediaQueryList | undefined => {
+    const matchMedia = window?.matchMedia;
+    return matchMedia && matchMedia('(prefers-color-scheme: dark)');
+};
+
+// in SSR, where window object is not defined, just return light theme
```

Comment:
we do not have SSR

Reply:
I wanted to preserve the already existing comment, but I think it's okay to remove it.